### DISCUSSION TITLE
constは変数じゃなくね？

### DISCUSSION THREAD

**Discussion by kituneponyo** [asciidwango/js-primer#1350] [質問]
URL : https://github.com/asciidwango/js-primer/blob/master/source/basic/introduction/README.md

> // `name`という名前の変数を宣言
> const name = "azu";
> // `NAME`という名前の変数を宣言
> const NAME = "azu";
> [:memo:](https://github.com/asciidwango/js-primer/edit/master/source/basic/introduction/README.md#L66 "Edit") <https://github.com/asciidwango/js-primer/blob/master/source/basic/introduction/README.md#L66>

**Comment by azu** [comment]
https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-declarations.html#sec-let-and-const-declarations

`const`は`variable`を定義する構文で、その`variable`は再代入できないという性質を持っています。
https://jsprimer.net/basic/variables/#const-is-not-constant
`const`で定義した`variable`に対して値は再代入できませんが、その値自体は変更することができるため、この書籍では定数という表現を避けています。(定数というと値自体が変わらないという認識を持つ人が多くいると考えたため)
そのため、この書籍では、`const`は「再代入できない変数」を定義する構文として解説しています。

なんという表現が適切だと思いますか?
